internal/utils/field/field.go

```diff
@@ -12,6 +12,22 @@ import (
 	"unicode"
 )
 
+func SetMapValues(input map[string]any, fieldName string, fieldValue any) {
```

Comment:
can we reuse SetValue  ?

Reply:
I don't think so. `SetValue` assumes that the input is a struct, this function assumes that the input is a `map[string]any`. 

It's possible to force `SetValue` to do this, but I don't think that this would make the code clearer.